src/main/resources/application.yaml

```diff
@@ -33,6 +33,7 @@ cantus:
 
 server:
     port: 8090
+    max-http-header-size: 200000
```

Comment:
Denne må være her for å støtte litt lengre query parametre. Noen som har innspill på hvor høy den burde være? nå støtter den å hente alle manifestene til Boober.